apps/dashboard/src/@3rdweb-sdk/react/hooks/useEngine.ts

```diff
@@ -1639,6 +1639,9 @@ export function useEngineCreateNotificationChannel(engineId: string) {
         `${THIRDWEB_API_HOST}/v1/engine/${engineId}/notification-channels`,
         {
           method: "POST",
+          headers: {
```

Comment:
This was missing but it was working for some reason?

Reply:
if the API u call is happy to respond with JSON even when not explicitly requested then it'll work